internal/git/git.go

```diff
@@ -1,27 +1,97 @@
-package git
+package main
 
 import (
+	"fmt"
+	"log"
+	"os"
+	"time"
+
 	"gopkg.in/src-d/go-git.v4"
+	"gopkg.in/src-d/go-git.v4/plumbing/object"
 )
 
 // Init returns a new repository using the .git folder, if the fixture
 // is tagged as worktree the filesystem from fixture is used, otherwise a new
 // memfs filesystem is used as worktree.
 func Init(path string) *git.Repository {
 
-	repo, err := git.PlainOpen(path)//
-		if err != nil {
-			//this feels wrong :-(
-			repo, initErr := git.PlainInit(path, false)
-			if initErr != nil {
-				panic(initErr)
-			}
-			return repo
+	repo, err := git.PlainOpen(path) //
+	if err != nil {
+		//this feels wrong :-(
+		repo, initErr := git.PlainInit(path, false)
+		if initErr != nil {
+			panic(initErr)
 		}
+		return repo
+	}
 	return repo
 
 }
 
+// CheckIfError should be used to naively panics if an error is not nil.
+func CheckIfError(err error) {
+	if err == nil {
+		return
+	}
+
+	fmt.Printf("\x1b[31;1m%s\x1b[0m\n", fmt.Sprintf("error: %s", err))
+	os.Exit(1)
+}
+
+// Info should be used to describe the example commands that are about to run.
+func Info(format string, args ...interface{}) {
+	fmt.Printf("\x1b[34;1m%s\x1b[0m\n", fmt.Sprintf(format, args...))
+}
+
+func clone(gitURL string, dir string) {
```

Comment:
functions need a capital letter to be "public".
clone -> Clone
commit -> Commit

Reply:
Thanks! I did not know. Comment addressed